I work with BlueAcorn on Store Fulfillment project (#1725)
magento/merchdocs: src/sales/order-workflow.md

When a customer places an order, a sales order is created as a temporary record of the transaction. In the Orders grid, sales orders initially have a status of "Pending," and can be canceled at any time until the payment is processed. After payment is confirmed, the order can be invoiced and shipped.

Step 1: Place Order
: The checkout process begins when the shopper clicks **Go to Checkout** on the shopping cart page or [reorders]({% link sales/reorders-allow.md %}) directly from their customer account.

Step 2: Order Pending
: The initial sales order status is `Pending`. In this state, the payment has not been processed and the order can still be edited or canceled. This will occur when the payment method is configured for authorization mode.

Step 3: Receive Payment
: The order status changes to `Processing` when payment is received or authorized. Depending on the payment method, you might receive notification when the transaction is authorized or processed. This will occur automatically when the payment method is configured for capture or intent sale mode.

Step 4: Invoice Order

> : An order is typically invoiced after payment is received. Some payment methods generate an invoice automatically when payment is authorized and captured. The payment method determines which invoicing options are needed for the order. After the invoice is generated and submitted, a copy is sent to the customer. This will occur automatically when payment processor module is configured for capture or intent sale mode.

Reviewer: ```suggestion
: An order is typically invoiced after payment is received. The payment method determines which invoicing options are needed for the order. After the invoice is generated and submitted, a copy is sent to the customer. If the payment method is configured with the `capture` or `intent sale` payment action, an invoice is generated automatically when payment is authorized and captured.
```

Step 3: Receive Payment
: The order status changes to `Processing` when payment is received or authorized. Depending on the payment method, you might receive notification when the transaction is authorized or processed. This will occur automatically when the payment method is configured for capture or intent sale mode.

Step 4: Invoice Order
: An order is typically invoiced after payment is received. The payment method determines which invoicing options are needed for the order. After the invoice is generated and submitted, a copy is sent to the customer. If the payment method is configured with the `capture` or `intent sale` payment action, an invoice is generated automatically when payment is authorized and captured.

Step 5: Book a Single Shipment
: The order status changes to to `Complete` when the shipment detail is complete, the shipment is booked, and either the packing slip and shipping label are printed or the 'Notify Ready for Pickup' button is clicked (if order was set for In-store delivery method.)  The customer receives notification and the package is shipped. If tracking numbers are used, the shipment can be tracked from the customer’s account.

{:.bs-callout-info}

> For details about order status and payment method configuration options, see [Order status]({% link sales/order-status.md %}) and [Payment Methods]({% link sales/payment-methods.md %}).

Reviewer: ```suggestion
For details about order status and payment method configuration options, see [Order status]({% link sales/order-status.md %}) and [Payment Methods]({% link configuration/sales/payment-methods.md %}).
```

: The checkout process begins when the shopper clicks the Go to Checkout button on the shopping cart page or [reorders]({% link sales/reorders-allow.md %}) directly from their customer account.

Step 2: Order Pending

> : The status of the sales order is initially `Pending`. Payment has not been processed, and the order can still be edited or canceled. This will occur when payment processor module is configured for authorization mode.

Reviewer: This will occur -- not sure what the last sentence means. What does "This" refer to?   Are you saying that an order only goes into Pending status for Payment methods that have been configured with the "authorization" action?  (See [Payment Methods - Payment actions]((https://docs.magento.com/user-guide/configuration/sales/payment-methods.html#payment-actions).

From the Adobe Commerce end-user perspective,  "payment processor module" the same as the Payment Method.

> : The status of the sales order is initially `Pending`. Payment has not been processed, and the order can still be edited or canceled. This will occur when payment processor module is configured for authorization mode.

Reviewer 2: ```suggestion
: The initial sales order status is `Pending`. In this state, the payment has not been processed and the order can still be edited or canceled. This will occur when payment processor module is configured for authorization mode.
```

Step 3: Receive Payment

> : The status of the order changes to `Processing` when payment is received or authorized. Depending on the payment method, you might receive notification when the transaction is authorized or processed. This will occur automatically when payment processor module is configured for capture or intent sale mode.

Reviewer 2: ```suggestion
: The order status changes to `Processing` when payment is received or authorized. Depending on the payment method, you might receive notification when the transaction is authorized or processed. This will occur automatically when the payment method is configured for capture or intent sale mode.
```

Reviewer 2: What is intent sale mode? I may just not be familiar with this term.

Step 4: Invoice Order

> : An order is typically invoiced after payment is received. Some payment methods generate an invoice automatically when payment is authorized and captured. The payment method determines which invoicing options are needed for the order. After the invoice is generated and submitted, a copy is sent to the customer. This will occur automatically when payment processor module is configured for capture or intent sale mode.

Reviewer: Last sentence:  What does "this" refer to -- the invoicing of the order?  or automatic invoicing?   Same comment regarding "payment processor module" = payment method

Reviewer 2: It seems to me you can combine the last two sentences for clarity....

Something like: "After the invoice is generated and submitted, a copy is automatically sent to the customer If the payment method is configured for capture or intent sale mode." or "If the payment method is configured for capture or intent sale mode, a copy of the invoice is automatically sent to the customer."

Also, if "capture" or "intent sale mode" are actually payment method statuses or configurations I have typically formatted those as code, as you do below with " `Complete` ". Unsure if that is needed here, but wanted to call it out. Is intent sale a mode? I guess I'm confused on this verbiage.

Step 5: Book a Single Shipment

> : The status of the order changes to `Complete` when the shipment detail is complete, the shipment is booked, and either the packing slip and shipping label are printed or the 'Notify Ready for Pickup' button is clicked (if order was set for In-store delivery method.)  The customer receives notification, and the package is shipped. If tracking numbers are used, the shipment can be tracked from the customer’s account.

Reviewer: ```suggestion
: The status of the order changes to `Complete` when the shipment detail is complete, the shipment is booked, and either the packing slip and shipping label are printed, or the 'Notify Ready for Pickup' button is clicked for orders scheduled for In-store delivery.  The customer receives notification, and the package is shipped. If tracking numbers are used, the shipment can be tracked from the customer’s account.
```

> : The status of the order changes to `Complete` when the shipment detail is complete, the shipment is booked, and either the packing slip and shipping label are printed or the 'Notify Ready for Pickup' button is clicked (if order was set for In-store delivery method.)  The customer receives notification, and the package is shipped. If tracking numbers are used, the shipment can be tracked from the customer’s account.

Reviewer 2: ```suggestion
: The order status changes to to `Complete` when the shipment detail is complete, the shipment is booked, and either the packing slip and shipping label are printed or the 'Notify Ready for Pickup' button is clicked (if order was set for In-store delivery method.)  The customer receives notification and the package is shipped. If tracking numbers are used, the shipment can be tracked from the customer’s account.
```
